1.Exploratory_Data_Analysis/exploratory_analysis.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
 #SBATCH --nodes=1
 #SBATCH --ntasks=1
-#SBATCH --mem=256G
+#SBATCH --mem=500G
 #SBATCH --partition=amem
 #SBATCH --qos=mem
 #SBATCH --time=24:00:00
 #SBATCH --output=sample-%j.out
 
 module load anaconda
 
-conda activate Interstellar
+conda activate Interstellar_python
 
-papermill 4.cell_count_analysis.ipynb 4.cell_count_analysis.ipynb -p celltype "SHSY5Y"
-papermill 1.umap_analysis_plate2.ipynb 1.umap_analysis_plate2.ipynb -p celltype "SHSY5Y"
+papermill notebooks/4.cell_count_analysis.ipynb noteboooks/4.cell_count_analysis.ipynb -p celltype "SHSY5Y"
+papermill notebooks/1.umap_analysis_plate2.ipynb notebooks/1.umap_analysis_plate2.ipynb -p celltype "SHSY5Y"
 
-papermill 4.cell_count_analysis.ipynb 4.cell_count_analysis.ipynb -p celltype "PBMC"
-papermill 1.umap_analysis_plate2.ipynb 1.umap_analysis_plate2.ipynb -p celltype "PBMC"
+#papermill notebooks/4.cell_count_analysis.ipynb notebooks/4.cell_count_analysis.ipynb -p celltype "PBMC"
+#papermill notebooks/1.umap_analysis_plate2.ipynb notebooks/1.umap_analysis_plate2.ipynb -p celltype "PBMC"
```

Comment:
Consider removing commented executions of other notebooks if you are no longer using them.